cookbook/bundles/best_practices.rst

```diff
@@ -221,7 +218,7 @@ following standardized instructions in your ``README.md`` file.
     following command to download the latest stable version of this bundle:
 
     ```bash
-    $ composer require <package-name> "~1"
+    $ composer require <package-name>
```

Comment:
I agree with you ... but at the same time, for DX reasons, we don't want to promote the use of versions in all cases. I propose to add a note explaining when do you need to add the version number. See 29fe426.